src/modules/Elsa.WorkflowTesting/Handlers/ConfigureRabbitMqActivitiesForTestHandler.cs

```diff
@@ -0,0 +1,62 @@
+using Elsa.Events;
+using Elsa.Testing.Events;
+using Elsa.WorkflowTesting.Services;
+using MediatR;
+
+namespace Elsa.WorkflowTesting.Handlers
+{
+    public class ConfigureRabbitMqActivitiesForTestHandler : INotificationHandler<WorkflowExecuting>, INotificationHandler<WorkflowFaulted>, INotificationHandler<WorkflowCompleted>, INotificationHandler<WorkflowTestExecutionStopped>
```

Comment:
This class should be moved to the `Elsa.Activities.RabbitMQ` project. The `Elsa.WorkflowTesting` module itself should be agnostic to specialized activities such as RabbitMQ (think open-closed principle where anyone should be able to add more activities to the system without the need for us to update core modules such as testing).